### PR TITLE
[1.5] Jmix Quartz: Simple Trigger issues

### PR DESCRIPTION
**Pull request by Gavrilov-Ivan**
See #1624

### CODE REVIEW

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/screen/trigger/trigger-model-edit.xml [hunk 1]
```diff
@@ -27,6 +27,9 @@
                 <comboBox id="scheduleTypeField" property="scheduleType" required="true" nullOptionVisible="false"/>
                 <textField id="cronExpressionField" property="cronExpression" required="true" visible="false"
                            contextHelpTextHtmlEnabled="true" contextHelpText="msg://cronExpressionHelpText"/>
+                <radioButtonGroup id="repeatModeSelector" optionsEnum="io.jmix.quartzui.screen.trigger.RepeatMode"
+                                  visible="false" caption="msg://triggerRepeatMode"
+                                  contextHelpText="msg://triggerRepeatModeHelpText" contextHelpTextHtmlEnabled="true"/>
                 <textField id="repeatCountField" property="repeatCount" visible="false"/>
```
Comment (gorbunkov) [inline jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/screen/trigger/trigger-model-edit.xml:33]: what about adding additional context help near the repeatCount field as well? We could explain one more time that "total number of firings will be this number + 1". Users may not read the context help near the radio group.

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties [hunk 2]
```diff
@@ -52,3 +52,14 @@ io.jmix.quartzui.screen.trigger/cronExpressionHelpText=Cron expression is a sequ
                     <li>0 11 11 11 11 ? - fires every November 11th at 11:11 AM</li>\
                     <li>0 0 0 25 12 ? - fires every Christmas at midnight</li>\
                 </ul>
+io.jmix.quartzui.screen.trigger/RepeatMode.NO_REPEATS=No repeats
```
Comment (gorbunkov) [inline jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties:55]: Maybe we can name this "Single execution" or "Run once" or "One-time trigger" or something like this? Won't these names better explain what will happen?

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties [hunk 3]
```diff
@@ -52,3 +52,14 @@ io.jmix.quartzui.screen.trigger/cronExpressionHelpText=Cron expression is a sequ
...
+io.jmix.quartzui.screen.trigger/RepeatMode.FINITE_REPEATS=Finite repeats
+io.jmix.quartzui.screen.trigger/RepeatMode.INFINITE_REPEATS=Infinite repeats
```
Comment (gorbunkov) [inline jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties:57]: "Run forever"? WDYT?

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties
Comment (gorbunkov) [inline jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties:1]: Please add new messagas to the jmix-translations for english and russian locales

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/screen/trigger/trigger-model-edit.xml (hunk 1 above)
Comment (Gavrilov-Ivan) [reply jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/screen/trigger/trigger-model-edit.xml]: Done

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties (hunk 2 above)
Comment (Gavrilov-Ivan) [reply jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties]: Changed

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties (hunk 3 above)
Comment (Gavrilov-Ivan) [reply jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties]: Changed

File: jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties
Comment (Gavrilov-Ivan) [reply jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties:1]: Done

**Review by gorbunkov — APPROVED** [review]

**Pull request merged**